package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "node-non-blocking-weather-lab",
+  "version": "1.0.0",
+  "description": "Weather App Lab",
+  "main": "index.js",
+  "scripts": {
+    "test": "node_modules/mocha/bin/mocha test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:learn-co-curriculum/node-non-blocking-weather-lab.git"
+  },
+  "author": "Azat Mardan",
+  "license": "ISC",
```

Comment:
Hm, this just occurred to me -- we should probably be careful to use the Learn license in these fields: http://learn.co/content-license